### ISSUE TITLE
[Deletion Request]  Delete Muslos no Sekai

### ISSUE THREAD

**Issue by Yura-ia** [tachiyomiorg/tachiyomi-extensions#18898]
### Source Websites

https://mangasnosekai.com/

### Other details

I come from the Scan "Muslos no sekai", I would like to have my website removed from here. It was added without my permission, so I would like you to delete my website from here, thanks. 

### Requirements

- [X] I've added a `TXT` DNS record for my domain with the value `tachiyomi-verification`
- [X] Site only hosts content scanlated by the group and not stolen from other scanlators or official releases (i.e., not an aggregator site)
- [X] Site is not infested with user-hostile features (e.g., invasive or malicious ads)

**Comment by vetleledaal** [comment]
>  I've added a `TXT` DNS record for my domain with the value `tachiyomi-verification`

Missing:
```bash
$ dig +short mangasnosekai.com TXT
"google-site-verification=yhKjaPlk3lNCH4UfhCJteKrqv6mFjedH8OtPaLcIv0o"
```

_


> Site only hosts content scanlated by the group and not stolen from other scanlators or official releases (i.e., not an aggregator site)

I found these after a quick glance:
```
Yotsuni Scanlation
AnimeBbg
Ryu no Fukkatsu
Ova no Fansub
```

**Comment by Yura-ia** [comment]
Of course, it's because in some cases we pick up from a chapter, but from there it's purely our work. Besides, when I uploaded their chapters I asked for their permission to do so.
